legacy/web/src/main/java/org/jboss/as/web/WebMigrateOperation.java

```diff
@@ -124,12 +130,12 @@ private WebMigrateOperation(boolean describe) {
     }
 
     static void registerOperations(ManagementResourceRegistration registry, ResourceDescriptionResolver resourceDescriptionResolver) {
-        registry.registerOperationHandler(new SimpleOperationDefinitionBuilder("migrate", resourceDescriptionResolver)
+        registry.registerOperationHandler(new SimpleOperationDefinitionBuilder(MIGRATE, resourceDescriptionResolver)
```

Comment:
The definition is missing the reply type (something like  a generic `.setReplyType(ModelType.PROPERTY).setReplyValueType(ModelType.OBJECT)`)